src/Bicep.Core/Syntax/SyntaxHelper.cs

```diff
@@ -27,5 +28,8 @@ public static class SyntaxHelper
 
             return allowedArraySyntax.Items.Select(i => i.Value);
         }
+
+        public static TypeSymbol? TryGetPrimitiveType(ParameterDeclarationSyntax parameterDeclarationSyntax)
+            => LanguageConstants.TryGetDeclarationType(parameterDeclarationSyntax.Type.TypeName);
```

Comment:
>TryGetDeclarationType [](start = 33, length = 21)

Does it make sense to have this function in `LanguageConstants`?

Reply:
Actually never mind. I think it does.

---
In reply to: [492991492](https://github.com/Azure/bicep/pull/546#discussion_r492991492) [](ancestors = 492991492)

Reply:
At some point we should probably name `LanguageConstants` something different or break it up.

Reply:
Yeah, future us problem :)

---
In reply to: [492998605](https://github.com/Azure/bicep/pull/546#discussion_r492998605) [](ancestors = 492998605)